easybuild/easyconfigs/p/phyx/phyx-1.3-foss-2022a.eb

```
@@ -0,0 +1,49 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'ConfigureMake'
+
+name = 'phyx'
+version = '1.3'
+
+homepage = 'https://github.com/FePhyFoFum/%(name)s'
+description = "phyx performs phylogenetics analyses on trees and sequences."
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True}
+
+github_account = 'FePhyFoFum'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s_link_flexiblas.patch']
+checksums = [
+    {'v1.3.tar.gz': 'b09677f98f997838b39cd2724463be06d0627ddc90a7e659dc90d4e02db624eb'},
+    {'phyx-1.3_link_flexiblas.patch': 'e9dc98da0f3c5df8e5b8ac3e64c2b0f77358f61fc60a4aadce5ad790b636ebdd'},
+]
+
+builddependencies = [
+    ('Autotools', '20220317', '', ('GCCcore', '11.3.0')),
```

```suggestion
    ('Autotools', '20220317'),
```

Shouldn't be any need to specify the toolchain here, just risk of breaking things for HMNS